### PR TITLE
Fixed cursor when duplicated header

### PR DESCRIPTION
**Pull request by jsakaicribl**
When processing duplicate headers with header auto, the position of meta.cursor will be returned as the position of cursor including renamed headers such as _<number>. But it's different from the cursor of the actual file. So set the cursor to the length of the original header.

### CODE REVIEW

File: papaparse.js [hunk 1]
```diff
@@ -1516,7 +1517,11 @@ License: MIT
 				for (var i = 0; i < rows.length; i++)
 				{
 					row = rows[i];
-					cursor += row.length;
+					if (config.header && i === 0 && firstLine !== undefined) {
```
Comment (pokoli) [inline papaparse.js:1520]: Do we really need to test if confix header? firstline will be undefined if config.header is not set, so I think we can remove the `config.header` test

Comment (pokoli) [inline papaparse.js:1520]: Can you add a comment like: 
```
// use firstline as row length may be changed due to duplicated headers
```
before the if clause

File: tests/test-cases.js [hunk 2]
```diff
@@ -2716,7 +2725,7 @@ describe('Custom Tests', function() {
 		const blob = new Blob([
 			`
 				importScripts('${papaParseScriptPath}');
-				
+
```
Comment (pokoli) [inline tests/test-cases.js:2728]: This seems unreleated to the change. Could you please avoid modifyinng unrelated lines? Thanks

**Review by pokoli — CHANGES_REQUESTED** [review]

File: papaparse.js (hunk 1 above)
Comment (jsakaicribl) [reply papaparse.js]: Thanks. I fixed that.

Comment (jsakaicribl) [reply papaparse.js]: Thanks. I fixed that.

**Comment by pokoli** [comment]
Your patch is merged, thanks for your contribution!
